CMakeLists.txt

```diff
@@ -98,7 +98,7 @@ endfunction()
 
 # Collect list of components to append to CMAKE_PREFIX_PATH
 list(APPEND _comps
-  bacio sigio sfcio gfsio ip sp w3nco g2 g2tmpl nemsio nemsiogfs landsfcutil w3emc wrf_io bufr ip2 wgrib2)
```

Comment:
Instead of removing it completely, can you put logic in if (FLAT) then remove, other wise build?

Reply:
you can also put big bold warning why you are removing it.